src/times.js

```diff
@@ -21,9 +21,13 @@ var _curry2 = require('./internal/_curry2');
  *      R.times(R.identity, 5); //=> [0, 1, 2, 3, 4]
  */
 module.exports = _curry2(function times(fn, n) {
-  var len = Number(n);
-  var list = new Array(len);
-  var idx = 0;
+  var len = Number(n),
+      idx = 0,
+      list;
+  if (len < 0 || isNaN(len)) {
+    throw new RangeError('n must be a non-negative number');
+  }
```

Comment:
i can vaguely imagine a scenario where this is getting evaluated lazily (or transduced?) where it would be nice to keep going one-by-one. ... vaguely. 

On another topic, `times` feels pretty imperative, much like `forEach`.